contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--2.5.0--2.6.0.sql

```diff
@@ -3,10 +3,189 @@
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
 
+-- Drops an object if it does not have any dependent objects.
+-- Is a temporary procedure for use by the upgrade script. Will be dropped at the end of the upgrade.
+-- Please have this be one of the first statements executed in this upgrade script. 
+CREATE OR REPLACE PROCEDURE babelfish_drop_deprecated_object(object_type varchar, schema_name varchar, object_name varchar) AS
+$$
+DECLARE
+    error_msg text;
+    query1 text;
+    query2 text;
+BEGIN
+
+    query1 := pg_catalog.format('alter extension babelfishpg_tsql drop %s %s.%s', object_type, schema_name, object_name);
```

Comment:
this should be babelfishpg_common otherwise you would get following error -- 
```
WARNING:  function binarysysvarchar_deprecated_2_6_0(bbf_binary) is not a member of extension "babelfishpg_tsql"
WARNING:  function binaryvarchar_deprecated_2_6_0(bbf_binary) is not a member of extension "babelfishpg_tsql"
WARNING:  function rowversionsysvarchar_deprecated_2_6_0(rowversion) is not a member of extension "babelfishpg_tsql"
WARNING:  function rowversionvarchar_deprecated_2_6_0(rowversion) is not a member of extension "babelfishpg_tsql"
```